en/docs/samples/auth-code-playground.md

```diff
@@ -0,0 +1,79 @@
+# Authorization Code Grant with OAuth Playground
+
+This page guides you through using a **sample Playground application** to try out authentication to an OAuth/OpenID Connect web application using the [Authorization Code](insertlink) grant type.
+
+----
+
+{! samples/oauth-playground.md !}
+
+----
+
+## Try Authorization Code grant 
+
+1.  Enter the following details.
+
+    - **Authorization Grant Type:** Authorization Code 
+    
+    - **Client ID:** The OAuth Client Key recieved when registering the service provider.
+    
+    - **Callback URL:** http://wso2is.local:8080/playground2/oauth2client
+    
+    - **Authorize Endpoint:** https://localhost:9443/oauth2/authorize
+    
+    - **Scope:** Any scope you wish to obtain the token for (e.g., `openid`). This field is optional. 
```

Comment:
Shall we specially mention that if someone needs to utilize the sample with OIDC they have to use 'openid' scope mandatorily?